app/views/articles/show.html.erb

```diff
@@ -1,7 +1,7 @@
 <div class="l-constrained">
-  <div class="hero">
+  <div class="hero" style="background-image: url(<%= article.hero_image_component %>);">
```

Comment:
can the article image be displayed with an image tag rather than mixing css in with html?

Reply:
I really wanted the same. I forgot to add on the description. I talked with @johnplayer1982 about it which he had a strong reasoning for it. @johnplayer1982 can you clarify again here, please?